scripts/release.ts

```diff
@@ -120,8 +121,9 @@ async function main() {
     const packageJsonPath = `${dir}/${packageName}/package.json`;
     let pkg = `${fs.readFileSync(packageJsonPath)}`;
     const parsedPkg = JSON.parse(`${pkg}`);
-    const latestVersion =
-        $(`npm view @tensorflow/${packageName} dist-tags.latest`);
+    const packageVersions: string[] =
+        JSON.parse($(`npm view @tensorflow/${packageName} versions --json`));
+    const latestVersion = semver.rsort(packageVersions)[0];
 
```

Comment:
Nit: Can we factor this and the other instance out into a `getVersions` or `getLatestVersion` function?

Reply:
We already have these functions (`getNpmVersion(..., tag='latest')`, etc) in `release-util.ts`, which are tag based and for `publish-npm.ts`. I'm not sure whether we should change all the latest tag based function to use maximum version number or not, it's hard to trace them all. I think the best approach is to only fix them if we actually see an error when running the script since some of them may actually depend the latest tagged version.